Specs/Scene/ModelExperimental/ModelExperimentalAnimationCollectionSpec.js

```diff
@@ -615,6 +616,100 @@ describe("Scene/ModelExperimental/ModelExperimentalAnimationCollection", functio
     });
   });
 
+  it("animates with an explicit animation time", function () {
+    return loadAndZoomToModelExperimental(
+      {
+        gltf: animatedTriangleUrl,
+      },
+      scene
+    ).then(function (model) {
+      const time = JulianDate.fromDate(
+        new Date("January 1, 2014 12:00:00 UTC")
+      );
+      const animationCollection = model.activeAnimations;
+      animationCollection.animateWhilePaused = false;
+      let animationTime = 0;
+      const animation = animationCollection.add({
+        index: 0,
+        animationTime: function (duration) {
+          return animationTime / duration;
+        },
+      });
+
+      const spyUpdate = jasmine.createSpy("listener");
+      animation.update.addEventListener(spyUpdate);
+
+      scene.renderForSpecs(time);
+      animationTime = 0.1;
+      scene.renderForSpecs(JulianDate.addSeconds(time, 1.0, scratchJulianDate));
+      scene.renderForSpecs(JulianDate.addSeconds(time, 2.0, scratchJulianDate));
```

Comment:
Add a comment here that this shouldn't trigger an update because the animation time hasn't changed.